src/backend/distributed/sql/udfs/citus_remote_connection_stats/latest.sql

```diff
@@ -0,0 +1,6 @@
+CREATE OR REPLACE FUNCTION citus_remote_connection_stats(OUT hostname text, OUT port int, OUT database_name text, OUT connection_count_to_node int)
+	RETURNS SETOF RECORD
+	LANGUAGE C STRICT
+	AS 'MODULE_PATHNAME', $$citus_remote_connection_stats$$;
+ COMMENT ON FUNCTION citus_remote_connection_stats(OUT hostname text, OUT port int, OUT database_name text, OUT connection_count_to_node int)
+     IS 'returns statistics about remote connections';
```

Comment:
does this need any REVOKE/GRANT?

Reply:
I guess, yes as it has performance implications and we should not allow users to call it liberally. Though, could be useful in debugging/looking deeper to the cluster.